in_toto/in_toto_sign.py

```diff
@@ -268,12 +269,12 @@ def create_parser():
       type=str, choices=SUPPORTED_KEY_TYPES,
       nargs="+", help=(
       "types of keys specified by the '--key' option. '{rsa}' keys are"
-      " expected in a 'PEM' format and '{ed25519}' in a custom"
+      " expected in a 'PEM' format. '{ed25519}' and '{ecdsa} in a custom"
```

Comment:
```suggestion
      " expected in a 'PEM' format. '{ed25519}' and '{ecdsa} are expected in a custom"
```

I think this goes over 80 cols so it needs wrapping.

Reply:
I've updated this now. Thanks!